src/apps/filters/h3ToLocalIj.c

```diff
@@ -47,32 +47,56 @@ void doCell(H3Index h, H3Index origin) {
     }
 }
 
-int main(int argc, char *argv[]) {
-    // check command line args
-    if (argc != 2) {
-        fprintf(stderr, "usage: %s [origin]\n", argv[0]);
-        exit(1);
-    }
+int main(int argc, char* argv[]) {
+    H3Index origin = 0;
+    H3Index index = 0;
 
-    H3Index origin;
+    Arg helpArg = {.names = {"-h", "--help"},
+                   .helpText = "Show this help message."};
+    Arg originArg = {
+        .names = {"-o", "--origin"},
+        .scanFormat = "%" PRIx64,
+        .valueName = "origin",
+        .value = &origin,
+        .required = true,
+        .helpText =
+            "Origin (anchoring index) for the local coordinate system."};
+    Arg indexArg = {.names = {"-i", "--index"},
+                    .scanFormat = "%" PRIx64,
+                    .valueName = "index",
+                    .value = &index,
+                    .helpText = "Index to convert to IJ coordinates."};
 
-    if (!sscanf(argv[1], "%" PRIx64, &origin))
-        error("origin could not be read");
+    Arg* args[] = {&helpArg, &originArg, &indexArg};
+    const int numArgs = 3;
+    const char* helpText = "Converts H3 indexes to local IJ coordinates";
```

Comment:
It's worth noting that the real documentation here is mostly in the `@brief` comment at the top of the file, where it will be inaccessible to the end user. Should we put more of that content into the `helpText` or into some other place where a user could read it without access to the code? In this case especially, there's a lot of useful info in the header comment.